src/Extensions/src/Eventuous.AspNetCore.Web/Http/ResultExtensions.cs

```diff
@@ -51,5 +60,18 @@ ActionResult AsProblemResult(int statusCode)
                 StatusCode   = Status400BadRequest,
                 ContentTypes = new MediaTypeCollection { ContentTypes.ProblemDetails },
             };
+
+        ActionResult AsValidationProblemResult(int statusCode)
+            => new ObjectResult(
+                new ValidationProblemDetails(new Dictionary<string, string[]> { ["Domain"] = [error.ErrorMessage] }) {
+                    Status = statusCode,
+                    Title  = error.ErrorMessage,
+                    Detail = error.Exception?.ToString(),
+                    Type   = error.Exception?.GetType().Name
+                }
+            ) {
+                StatusCode   = Status400BadRequest,
+                ContentTypes = new MediaTypeCollection { ContentTypes.ProblemDetails },
+            };
```

Comment:
There seems to be quite a lot of repetition in code, does it mean that extensions or something don't work?

Reply:
Well, that is what I thought as well. For me it was not clear if some things were done on purpose. 
1. the private methods as sub methods (for an example `AsProblemResult`)
2. `CommandHttpApiBase` and `CommandHttpApiBase<T>` aren't following the `ProblemDetails` setup and weren't using the extensions.

But all methods are slightly different so I'm not sure if we could write 1 extension method for everything. Maybe do this in separate commit?